Testscripts/Linux/utils.sh

```diff
@@ -3763,4 +3762,23 @@ function Check_lsvmbus()
 		update_repos
 		install_package python
 	fi
+}
+
+# Check if this VM is IB over ND, IB over SR-IOV or non-HPC VM.
+# Return 0, if VM is IB over ND.
+# Return 1, if VM is IB over SR-IOV.
+# Return 2, if VM is non HPC VM.
+# Return 3, if unknown.
+function is_hpc_vm() {
+	if [ -d /sys/class/infiniband/ ]; then
+		if [ -n "$(lspci | grep "Virtual Function")" ] && [ -n "$(dmesg | grep "IB Infiniband driver")" ]; then
+			return 1
+		elif [ -n "$(dmesg | grep hvnd_try_bind_nic)" ]; then
+			return 0
+		else
+			return 3
```

Comment:
When can this (unknown) happen ?

Reply:
This should be a buggy situation like HPC VM does not load VF interface or ND driver properly. It should abort the testing.

Reply:
nit: return sequence 1,0,3 ,2 looks odd.. can the sequence be rearranged/renamed to return 0,12,3